CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ option(NGRAPH_NOP_ENABLE "Control the building of the NOP backend" TRUE)
 option(NGRAPH_DISTRIBUTED_ENABLE "Add distributed mode to the CPU backend" FALSE)
 option(NGRAPH_DEBUG_ENABLE "Enable output for NGRAPH_DEBUG statements" FALSE)
 option(NGRAPH_ONNX_IMPORT_ENABLE "Enable ONNX importer" FALSE)
-option(NGRAPH_DEX_ONLY "Build CPU DEX without codegen" FALSE)
+option(NGRAPH_DEX_ONLY "Build CPU or GPU DEX without codegen" FALSE)
```

Comment:
for my understanding, DEX will be the only way we need for GPU. did I miss something?

Reply:
Yes, but our codegen version currently supports things that the new DEX impl. needs, like propogate inplace input/output and performance timers. 

Until these are added and we've tested DEX to be fully implemented, codegen is default and DEX can be used if you set `export NGRAPH_CODEGEN=0` or if you compile with NGRAPH_DEX_ONLY=TRUE.